public/apidocs/swagger_v3.json

```diff
@@ -2061,7 +2061,7 @@
             "type": "string"
           }
         },
-        "funding": {
+        "mechanism": {
```

Comment:
funding was an array of strings, but mechanism is only a string. look at the "candidate" property to see how it should be written

also, see if you can move "mechanism" up to below "candidate" (i mindlessly changed the order of the properties the API returns). here's sample output from production to see the order:
```json
    "data": [
        {
            "candidate": "Ad5-nCoV",
            "mechanism": "Recombinant vaccine (adenovirus type 5 vector)",
            "sponsors": [
                "CanSino Biologics"
            ],
            "details": "Background: China&rsquo;s CanSino Biologics has developed a recombinant novel coronavirus vaccine that incorporates the adenovirus type 5 vector (Ad5). Preliminary safety data from a Phase 1 (ChiCTR2000030906; NCT04313127) clinical trial of 108 participants between 18 and 60 years old who will receive low, medium, and high doses of Ad5-nCoV has allowed the company to plan to initiate a Phase 2 trial, according to an announcement. The Phase 2 (ChiCTR2000031781) trial has identical inclusion criteria. Outcomes: In animal studies, a single dose of Ad5-nCoV protected against upper respiratory infection of SARS-CoV-2 in ferrets, according to a paper published 14 August in Nature Communications. Results from Phase 1 of the trial show a humoral and immunogenic response to the vaccine, according to a paper published in The Lancet. Adverse reactions such as pain (54%), fever (46%), fatigue (44%), headache (39%), and muscle pain (17%) occurred in 83% of patients in the low and medium dose groups and 75% of patients in the high dose group. In Phase 2 of the trial, neutralizing antibodies and specific interferon γ enzyme-linked immunospot assay responses were observed at all dose levels for most participants. Status: On 25 June, China’s Central Military Commission announced the military had been approved to use Ad5-nCoV for a period of 1 year, according to reporting in Reuters. A Phase 3 trial in Saudi Arabia is currently underway.",
            "trialPhase": "Phase 3",
            "institutions": [
                "Tongji Hospital",
                "Wuhan, China"
            ]
        },
```

Reply:
Got it, thanks for the sample output!